workflows/nallo.nf

```diff
@@ -205,96 +197,228 @@ workflow NALLO {
         //
         // Check sex and relatedness, and update with infered sex if the sex for a sample is unknown
         //
-        BAM_INFER_SEX ( bam_infer_sex_in, fasta, fai, ch_somalier_sites, ch_samplesheet_pedfile )
+        BAM_INFER_SEX (
+            bam_infer_sex_in,
+            fasta,
+            fai,
+            ch_somalier_sites,
+            ch_samplesheet_pedfile
+        )
         ch_versions = ch_versions.mix(BAM_INFER_SEX.out.versions)
-
         ch_multiqc_files = ch_multiqc_files.mix(BAM_INFER_SEX.out.somalier_samples.map{it[1]}.collect().ifEmpty([]))
         ch_multiqc_files = ch_multiqc_files.mix(BAM_INFER_SEX.out.somalier_pairs.map{it[1]}.collect().ifEmpty([]))
 
+        // Set files with updated meta for subsequent processes
         bam     = BAM_INFER_SEX.out.bam
         bai     = BAM_INFER_SEX.out.bai
         bam_bai = BAM_INFER_SEX.out.bam_bai
 
-        //
-        // Create PED with updated sex per project (should perphaps be per SNV-calling region)
-        //
+    }
+
+    //
+    // Run read QC with FastQC, mosdepth and cramino
+    //
+    if (!params.skip_mapping_wf && !params.skip_qc) {
```

Comment:
I'm a bit confused why we need these double checks. Does the pipeline not ensure consistency before?

For instance, here:

|                           |          | `params.skip_mapping_wf`  |  |
| - | - | - | - |
|                           |          | True | False |
| `params.skip_qc` | True | Skips QC | Skips QC |
|                            | False | Doesn't run at all | Runs QC |

So, in the end, whether we run QC or not only depends on the state of `skip_qc`

Reply:
Yeah, I debated whether to just leave `!params.skip_qc`. I thought I kept them for readability, but maybe it becomes messier with the double checks?

Reply:
I think it would be clearer with checking just one parameter (or at least stating the parameter specific to the current workflow first)